packages/cli-kit/src/public/node/ui.tsx

```diff
@@ -686,14 +686,18 @@ export const keypress = async (stdin = process.stdin, uiDebugOptions: UIDebugOpt
   })
 }
 
+export function isTTY(stdin: NodeJS.ReadStream | undefined = undefined, uiDebugOptions: UIDebugOptions = defaultUIDebugOptions) {
+  return uiDebugOptions.skipTTYCheck || stdin || terminalSupportsRawMode()
+}
+
```

Comment:
Is there an existing utility for this? 

We expect folks to pass input into this method programmatically in regular use, so I don't want the command to crash out if they're not in an interactive terminal.